components/arm/trossen/trossen.go

```diff
@@ -239,6 +254,83 @@ func (a *Arm) JointPositions(ctx context.Context, extra map[string]interface{})
 	return &pb.JointPositions{Values: positions}, nil
 }
 
+// DoCommand handles incoming gripper requests.
+func (a *Arm) DoCommand(ctx context.Context, cmd map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
[major] do we actually need this command? Can we call the functions from  `arm.Local.(*trossenArm).OpenGripper()` interface eventually in the gripper?

An alternative choice is to just move the trossen gripper code into the arm package (keeping the validation and resource name constant). Up to you.

Reply:
If that's too wrapped up then ignore me.

Reply:
I think a DoCommand was a good add anyway because it allows a user to maybe not have to add a separate trossen gripper if they do not want to. Though this may be an argument for removing the gripper component altogether, I think that would be classified as a breaking change.